### DISCUSSION TITLE
install nvidia driver on Ubuntu 20.04 image

### DISCUSSION THREAD

**Discussion by ghost** [actions/virtual-environments#10683] [General questions]
Which version of nvidia driver is compatible with the following version of image?
[https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20240915.1](url)

**Comment by vidyasagarnimmagaddi** [comment]
> Which version of nvidia driver is compatible with the following version of image? [https://github.com/actions/runner-images/releases/tag/ubuntu20%2F20240915.1](url)

Hi @jinweig , Kindly follow the [announcement](https://github.com/actions/runner-images/issues/11101) , Ubuntu 20 will not available from 2025-04-01 onwards, hence we cannot include nvidia drive in Ubuntu 20 image
